eman2/protocols/__init__.py

```diff
@@ -35,3 +36,11 @@
 from .protocol_tiltvalidate import EmanProtTiltValidate
 from .protocol_autopick_boxer import EmanProtAutopick
 from .protocol_autopick_sparx import SparxGaussianProtPicking
+
+from .tomo_protocols import *
```

Comment:
This could go away too.

Reply:
Actually there is a duplication here.

Reply:
Yep, didn't notice that. Fixed in https://github.com/scipion-em/scipion-em-eman2/commit/3b805869019181e3e364f9837a73ef99f09c12dd Again, we may want to move the tomo protocols outside the sub folder